chroot-bin/runsommelier

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -eu
+# Copyright (c) 2021 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+CLUTTER_BACKEND=wayland
+DISPLAY=:0
+GDK_BACKEND=x11
+SCALE=1
```

Comment:
should this also be user-overridable?

Reply:
commit 0b040a4c706d90293497c9011e77fcc41b92ae59